### DISCUSSION TITLE
[New Feature] Deployed structures can't be sold other than the Construction Yard.

### DISCUSSION THREAD

**Discussion by MikillRosen** [Vinifera-Developers/Vinifera#1458] [Feature Requests]
### Description:
It seems the game hardcoded structures that can be undeployed other than the Construction Yard to vehicles to not be sellable even if you set them to be sellable.

### Possible Implementation:
Remove this hardcode and make `Unsellable` default to `yes` for structures that can undeploy to vehicles.

**Comment by CCHyper** [comment]
I recall having a discussion with @Bittah about this issue a long time ago. If I remember correctly, we came to the conclusion that a new INI key for BuildingTypes such as `ForceSellable=` would be ideal, leaving the original logic untouched.
